list_lookup_test.go

```diff
@@ -0,0 +1,358 @@
+package gotwtr_test
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sivchari/gotwtr"
+)
+
+func Test_lookUpOwnedListByID(t *testing.T) {
```

Comment:
add `t.Parallel()`

Reply:
I think `t.Parallel()` is already writing